tests/clv/models/test_beta_geo.py

```diff
@@ -160,63 +118,133 @@ def test_expected_probability_alive(self):
         The "true" prefix refers to the value obtained using 1) the closed form
         solution and 2) the data-generating parameter values.
         """
-        true_prob_alive = self.alive.mean()  # scalar
-        est_prob_alive = self.fixed_model.expected_probability_alive(
-            self.customer_id,
-            self.frequency,
-            self.recency,
-            self.T,
+        rng = np.random.default_rng(152)
+
+        N = 100
+        # Almost deterministic p = .02
+        a = 0.02 * 10_000
+        b = 0.98 * 10_000
+        alpha = 3
+        r = 4
+
+        recency, frequency, alive, T = self.generate_data(a, b, alpha, r, N, rng=rng)
+        customer_id = list(range(N))
+
+        bg_model = BetaGeoModel(
+            customer_id=customer_id,
+            frequency=frequency,
+            recency=recency,
+            T=T,
+            a_prior=pm.Normal.dist(a, 1e-3),
+            b_prior=pm.Normal.dist(b, 1e-3),
+            alpha_prior=pm.Normal.dist(alpha, 1e-3),
+            r_prior=pm.Normal.dist(r, 1e-3),
```

Comment:
Why the informative priors here?

Reply:
For a fake posterior based on prior_predictive sampling around the true values

Reply:
Replaced by direct sampling with numpy when creating the fake fit